cep-sigstore-predicate.md

```diff
@@ -0,0 +1,303 @@
+      English sentence, producing her attestation:
+
+        > Alice published the `widgets` package at version v1.2.3 with
+        > hash `sha256:abcd...` to the `conda-forge` channel.
+
+    - Bob establishes trust in Alice's public key.
+    - Bob can verify the attestation's signature against Alice's public key,
+      giving him confidence that the statement is true.
+    - Correspondingly, Bob can reject any statement for `widgets` that is not
+      signed by Alice's public key.
+
+- [in-toto] is a framework and standard for defining attestations.
+
+    - Within in-toto, an attestation's statement is composed of a
+      **subject** and a **predicate**. The subject is the resource
+      (or resources) being attested to, and the predicate is a
+      an arbitrary collection of metadata about the subject.
+      The predicate is identified by a **predicate type**,
+      which defines the predicate's expected schema.
+
+- [Sigstore] is a project that enables misuse-resistant software signing
+  and verification via short-lived certificates and a tamper-evident log.
+  Sigstore composes with attestation frameworks like in-toto to provide
+  transparency and misuse-resistance properties on top of the integrity
+  and authenticity properties of attestations.
+
+   One of Sigstore's major misuse-resistance contributions is
+   the use of *ephemeral keys* for signing. Modifying the example above:
+
+   - Instead of maintaining a long-lived signing key, Alice generates an
+     *ephemeral key* and binds it to her *identity*
+     ("`alice@trustme.example.com`").
+
+     This binding is done via a certificate issued by [Fulcio], which verifies a
+     *proof of possession* (such as from [OpenID Connect]) from Alice for her
+     claimant identity. The certificate issued by Fulcio is, in turn auditable
+     via [RFC 6962] Certificate Transparency (CT) logs.
+
+    - Alice signs her attestation with her ephemeral key, and distributes a
+      "bundle" containing both her attestation and her signing certificate.
+
+    - Instead of establishing trust a long-lived key from Alice, Bob establishes
+      trust in Alice's identity.
+
+    - Bob can verify the attestation's signature against Alice's emphemeral key,
+      which in turn can be verified as authentically Alice's via the Fulcio-
+      issued certificate.
+
+    With this flow, neither Alice nor Bob needs to maintain long-lived signing
+    or verifying keyrings, in turn reducing the attacker surface for key
+    compromise.
+
+  Another key misuse-resistance contribution within Sigstore is *machine
+  identities*. A machine identity behaves similarly to a human identity
+  (Alice or Bob), but identifies a machine instead of a human. For example,
+  `github.com/example/example/.github/workflows/release.yml@refs/tags/v1.2.3`
+  could be the machine identity of a GitHub Actions workflow that ran from
+  `release.yml` within `example/example` against the `v1.2.3` tag.
+
+## Motivation
```

Comment:
The motivation still seems *mostly* catered toward "attestations in general" and makes very little mention that this is "attesting about the publication metadata".  I wonder if this could be slightly reworded to emphasize the actual goals of `publish/v1`

Reply:
Any suggestions how to reword it?